packages/core/src/prompts/prompt.ts

```diff
@@ -194,10 +195,15 @@ export default class Prompt {
 		if (char && (char.toLowerCase() === 'y' || char.toLowerCase() === 'n')) {
 			this.emit('confirm', char.toLowerCase() === 'y');
 		}
-		if (char === '\t' && this.opts.placeholder) {
+		if ((char === '\t' || key?.name === 'return') && this.opts.placeholder) {
 			if (!this.value) {
-				this.rl?.write(this.opts.placeholder);
-				this.emit('value', this.opts.placeholder);
+				if (this.opts.defaultValue) {
```

Comment:
we don't need to add `defaultValue` as the individual prompts already deal with that

at some point it makes sense to refactor so we set it in one place i think ,but we don't need to do it yet

if you can add a test for this too, that'd be super helpful (prompt already has some tests)